autopush/router/apnsrouter.py

```diff
@@ -3,6 +3,7 @@
 
 from twisted.logger import Logger
 from twisted.internet.threads import deferToThread
+from hyper.http20.exceptions import ConnectionError, HTTP20Error
```

Comment:
the hyper imports should go above twisted's in all 3 files.. (flake8-isort coming soon)